### ISSUE TITLE
Make copy changes to 10-7959a per latest Figma

### ISSUE THREAD

**Issue by michaelclement** [department-of-veterans-affairs/va.gov-team#98936]
Per latest [Figma](https://www.figma.com/design/Tfhq5h2LwXEeEEtFBAAFOv/CHAMPVA-Claims-(10-7959a)?node-id=2913-117921&node-type=canvas&t=MftJqR7toK4Mn7gl-0), we need the following copy changes:

- [x] Update intro page with bold text for ‘1 year’
- [x] Update sponsor name title and hint text (note that it’s different for sponsors vs beneficiaries/3rd parties)
- [x] Add extra line “You can add up to 2...” on hint on /insurance-status screen
- [x] Update header and hint on /claim-auto-accident to say “car” instead of “auto” etc
- [x] Update the expandable info section header on /medical-claim-upload, /eob-upload, and - OHIs upload screen, and /pharmacy-claim-upload
- [x] Add the word “medication” to the prescription/pharmacy path upload (/pharmacy-claim-upload)

Acceptance criteria:
- The above copy changes are present on staging

**Comment by michaelclement** [comment]
[Parent issue](https://github.com/department-of-veterans-affairs/va.gov-team/issues/98798)

**Comment by michaelclement** [comment]
Waiting on platform before this can be merged into staging

**Comment by baltes** [comment]
@michaelclement Noticed some minor things:
- [ ] Add a space before second instance of "1 year"
- [ ] On signer type page change 'dependant' to 'dependent'
- [ ] On sponsor info page change 'dependant' to 'dependent' in hint text

**Comment by michaelclement** [comment]
| | |
|-|-|
|![Image](https://github.com/user-attachments/assets/11c13815-d4ec-4268-90ee-6188880eac95)|![Image](https://github.com/user-attachments/assets/b874c89e-6250-4777-a949-c12e6035e5b5)|
|![Image](https://github.com/user-attachments/assets/7c0cf2fd-0fa4-4c72-bb24-992e2a2b0bc3)| |

**Comment by michaelclement** [comment]
@baltes Updates added, pending platform review. Thanks!
